anitya/app.py

```diff
@@ -38,22 +37,46 @@
 APP.oid = OpenID(APP)
 
 # Set up the logging
-if not APP.debug:
-    APP.logger.addHandler(anitya.mail_logging.get_mail_handler(
-        smtp_server=APP.config.get('SMTP_SERVER', '127.0.0.1'),
-        mail_admin=APP.config.get('MAIL_ADMIN', 'admin@fedoraproject.org')
+logging_config = {
```

Comment:
While that does allow for maximum configuration, when I wrote this I thought that maybe it's more user-friendly to have one log level setting. I don't mind either way so if you think it's better to expose all this in the config I can switch it.